src/rnaseq-pipeline.nf

```diff
@@ -86,4 +90,4 @@ process cufflinks {
         cufflinks -p $params.cores -G $params.ref_gene -b $params.ref_fasta -L experiment_descriptor -u $bam_file
         """
         
-}
+}
```

Comment:
As a general rule of thumb, text files containing code must be terminated by a newline.